### DISCUSSION TITLE
Reset app

### DISCUSSION THREAD

**Discussion by JoshAtticus** [pear-devs/pear-desktop#503] [Q&A]
I accidentally hid the menu and now I can't show it. I've tried both alt keys and the esc key. I tried deleting the file and redownloading but the menu won't show.

**Comment by Araxeus** [comment]
See https://github.com/th-ch/youtube-music/issues/410#issuecomment-952060709

To change the config.json file

**Comment by Cszyn** [comment]
I also want to reset the app, not that I have any particular problems, but I just want to do it
